docs/containers-registries.d.5.md

```diff
@@ -61,6 +61,11 @@ more general scopes is ignored.  For example, if _any_ configuration exists for
 `docker.io/library/busybox`, the configuration for `docker.io` is ignored
 (even if some element of the configuration is defined for `docker.io` and not for `docker.io/library/busybox`).
 
+### Built-in Defaults
+
+If default-docker not configured and no matched configuration applied for the container image can be found,
```

Comment:
(Maybe something like
```suggestion
If no `docker` section can be found for the container image, and no `default-docker` section is configured
```
to go from the most-specific to the most general, in the “order of evaluation”)